0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -1,73 +1,56 @@
+* **SPSP Endpoint** - The specific HTTPS endpoint on the SPSP Server used for setting up a payment
+* **STREAM Module** - Software included in the SPSP client and server that implements the [STREAM](../0029-stream/0029-stream.md) protocol.
 
 ### Relation to Other Protocols
 
-SPSP is used for exchanging payment information before an ILP payment is initiated. The sender and receiver use the [STREAM](../0029-stream/0029-stream.md) transport protocol to generate the ILP packets. The receiver generates the shared secret and ILP address to be used in STREAM and communicates it to the sender over HTTPS.
-
-### Model of Operation
-
-We assume that the sender knows the receiver's SPSP endpoint (see [Payment Pointers](../0026-payment-pointers/0026-payment-pointers.md)).
-
-1. The sender's SPSP Client queries the receiver's SPSP Endpoint.
-2. The SPSP Endpoint responds with the receiver info, including the receiver's ILP address and the shared secret to be used in STREAM. It MAY respond with a balance associated with this SPSP receiver, i.e. in the case of an invoice.
-3. The sender constructs an ILP payment using the receiver's ILP address.
-4. The sender begins sending the payment.
-    1. The sender uses STREAM to establish a ILP/STREAM connection to the receiver.
-    2. The sender will construct a stream to the receiver on the ILP/STREAM connection.
-    3. The sender will adjust their `sendMax` to reflect the amount they're willing to send.
-    4. The receiver will adjust their `receiveMax` to reflect the amount they're willing to receive.
-    5. The sender's and receiver's STREAM modules will move as much value as possible while staying inside these bounds.
-    6. If the sender reaches their `sendMax`, they end the stream and the connection. If the receiver reaches their `receiveMax`, they will end the STREAM and the connection.
+SPSP is used for exchanging payment information before an ILP payment is initiated. The client and server use the [STREAM](../0029-stream/0029-stream.md) transport protocol to generate the ILP packets. The server generates the shared secret and ILP address to be used in STREAM and communicates it to the client over HTTPS.
 
 ## Specification
 
-The SPSP endpoint is a URI used by the sender to query information about the receiver (which may be an invoice) and set up payments. The SPSP endpoint URI MUST NOT contain query string parameters. The sender SHOULD treat the URI as opaque. There are several supported ways to refer to an SPSP endpoint:
+The SPSP endpoint is a URL the SPSP payment pointer resolves to, used by the client to query information about the server and set up payments. Clients MUST NOT send query string parameters in requests to the SPSP endpoint URL. Servers that receive query string parameters in an SPSP request MUST reject the request with a 400 Bad Request HTTP response code. Clients SHOULD treat the URL as opaque and not depend on any information they derive from the URL.
```

Comment:
I can change that. In the original spec it was not consistently capitalized so I decided to drop all the capitalization. If you prefer to have it capitalize, I would suggest capitalizing it everywhere.

Reply:
The important change is dropping the SPSP.

I think we should capitalise defined terms but I say that because it makes it clearer to me not necessarily because it "the right thing to do". Up to you

Reply:
> The important change is dropping the SPSP.

Should it be dropped [here](https://github.com/sabinebertram/rfcs/blob/basic-spsp/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md#interfaces) as well?